README.md

```diff
@@ -3,6 +3,7 @@
 [![PyPI version](https://badge.fury.io/py/ndexchange.svg)](https://badge.fury.io/py/ndexchange)
 [![Build Status](https://travis-ci.org/neurodata/ndex.svg?branch=master)](https://travis-ci.org/neurodata/ndex)
 [![Coverage Status](https://coveralls.io/repos/github/neurodata/ndpull/badge.svg?branch=master)](https://coveralls.io/github/neurodata/ndpull?branch=master)
+![Downloads shield](https://img.shields.io/pypi/dm/ndio.svg)
```

Comment:
this is the wrong package -this is ndexchange :)

https://img.shields.io/pypi/dm/ndexchange.svg